src/py/lib/pystone.py

```diff
@@ -183,17 +184,17 @@ def Proc3(PtrParOut):
 
     if PtrGlb is not None:
         PtrParOut = PtrGlb.PtrComp
+        PtrGlb.IntComp = Proc7(10, IntGlob)
     else:
         IntGlob = 100
-    PtrGlb.IntComp = Proc7(10, IntGlob)
```

Comment:
PtrGlb must not be None if you are assigning an attribute to it.

Reply:
This file is an odd one. We only use it in `benchmarks.py` I wonder if we could replace it with something else that's less like... whatever this file is like. I think of it as an external file, so I have avoided doing any sort of maintenance on it.

Reply:
We can always add files to the exclude list if you would rather, potentially for all of pre-commit checks (though mypy is a little harder, as if it's imported from a file we are checking, it has to also be handled there).

Reply:
Yeah, it should be added to the exclude list. It's a copy from https://github.com/blackberry/Python/blob/master/Python-3/Lib/test/pystone.py

Reply:
Added this file and the extern directory to the universal exclude list.